src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -45,10 +45,16 @@ enums:
         meaning: rdfs:Class
       rdfs literal:
         meaning: rdfs:Literal
+        description: This value indicate that the entity being mapped is not a semantic entity with a distinct identifier, but is instead represented entirely by its literal label. This value MUST NOT be used in the predicate_type slot.
```

Comment:
Its very easy to add the predicate_type restriction to the model directly (using `none_of`). Shall I do this for completeness or is the spec comment enough? (Expecting that tool implementors have to add that manually)

Reply:
I’d rather do that as a separate PR in which we also do that for the other `entity_type` values that should not be used in `predicate_type`, but no objection.

Reply:
I agree, lets not scope creep this PR! If we want to do this formally, we do it in the next iteration. We have already documented it, so its not a "schema change" when we then do it.